apps/communications/contacts/js/contacts.js

```diff
@@ -23,14 +24,8 @@ var COMMS_APP_ORIGIN = location.origin;
 var SCALE_RATIO = window.devicePixelRatio || 1;
 
 var Contacts = (function() {
-  var SHARED = 'shared';
-  var SHARED_PATH = '/' + SHARED + '/' + 'js';
 
-  var SHARED_UTILS = 'sharedUtilities';
-  var SHARED_UTILS_PATH = SHARED_PATH + '/contacts/import/utilities';
-
-  var SHARED_CONTACTS = 'sharedContacts';
-  var SHARED_CONTACTS_PATH = SHARED_PATH + '/' + 'contacts';
+  var SHARED_UTILS_PATH = '/shared/js/contacts/import/utilities';
```

Comment:
Why dont we remove this? Let's use the complete path, or use something as `Loader.sharedPath` (due to this is replicated in `Loader`) for retrieving it (exposing it as a `get` would be fine).